.platform.app.yaml

```diff
@@ -42,16 +44,14 @@ relationships:
     database: "database:mysql"
 
 web:
-    locations:
-        '/':
-            # The public directory of the application relative to its root.
-            root: 'web'
-            # The front-controller script which determines where to send
-            # non-static requests.
-            passthru: '/index.php'
```

Comment:
You probably still want the `locations` block, so that static assets are served as you expect. Just replace `passthru: '/index.php'` with `passthru: true` (once your application is talking HTTP directly, it has no concept of script anymore).